lmdk/modules/down_mixer/downmixer.c

```diff
@@ -0,0 +1,520 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Lets not duplicate the module sources in the code base, we should use the same module code regardless of basefw or module/library build.